tests/ballerina-test/src/test/resources/test-src/connectors/sql/sql_actions_test.bal

```diff
@@ -505,6 +571,18 @@ function testOutParameters() returns (any, any, any, any, any, any, any, any, an
     sql:Parameter paraSmallInt = { sqlType: sql:TYPE_SMALLINT, direction: sql:DIRECTION_OUT };
     sql:Parameter paraClob = { sqlType: sql:TYPE_CLOB, direction: sql:DIRECTION_OUT };
     sql:Parameter paraBlob = { sqlType: sql:TYPE_BLOB, direction: sql:DIRECTION_OUT };
+    if (jdbcUrl.contains("postgres")) {
```

Comment:
Avoid these postgres check and move all blob tests to a separate test. Check all the places.

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/8801 to address this. Will fix separately.